buf.gen.yaml

```diff
@@ -2,7 +2,10 @@ version: v1
 plugins:
   - name: go
     out: internal
+    opt:
+      - paths=source_relative
   - name: go-grpc
     out: internal
     opt:
-      - require_unimplemented_servers=false
\ No newline at end of file
+      - require_unimplemented_servers=false
+      - paths=source_relative
```

Comment:
We need `source_relative` here, otherwise we get:
```
$ tree internal

internal
├── github.com
│  └── openmarketplaceengine
│     └── openmarketplaceengine
│        └── internal
│           └── omeapi
│              ├── location
│              │  └── v1beta1
│              │     ├── location.pb.go
│              │     └── location_grpc.pb.go
│              ├── tollgate
│              │  └── v1beta1
│              │     ├── tollgate.pb.go
│              │     └── tollgate_grpc.pb.go
│              ├── tollgate_crossing
│              │  └── v1beta1
│              │     ├── tollgate_crossing.pb.go
│              │     └── tollgate_crossing_grpc.pb.go
│              └── worker
│                 └── v1beta1
│                    ├── worker.pb.go
│                    └── worker_grpc.pb.go
```